src/write_outputs/capacity_reserve_margin/effective_capacity.jl

```diff
@@ -1,6 +1,6 @@
 @doc raw"""
-	thermal_plant_effective_capacity(EP::Model,
-	                                 inputs::Dict,
+    thermal_plant_effective_capacity(EP::Model,
+                                     inputs::Dict,
                                      resources::Vector{Int},
                                      capres_zone::Int,
                                      timesteps::Vector{Int})::Matrix{Float64})
```

Comment:
This is from formatting with JuliaFormatter. I'm not sure why this didn't get picked up before.